src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -113,10 +102,6 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.AddTransient<VirtualFileSystem>();
             services.AddTransient<VirtualFileSystemMiddleware>();
 
-            // we want all ILoggerFactory resolution to go through WebHostResolver
-            // TODO: DI (FACAVAL) This is no longer the case... perform cleanup (/cc brettsam)
-            // builder.Register(ct => ct.Resolve<WebHostResolver>().GetLoggerFactory(ct.Resolve<WebHostSettings>())).As<ILoggerFactory>().ExternallyOwned();
```

Comment:
`LoggerFactory` is now just follows the standard injection/configuration pattern.